lib/v4/errorMap/errorMap.test.ts

```diff
@@ -1,38 +1,6 @@
 import * as zod from 'zod/v4';
-import { vi } from 'vitest';
-import {
-  createErrorMap,
-  defaultErrorMap,
-  defaultErrorMapOptions,
-} from './errorMap.ts';
 
-describe('errorMap', () => {
-  test('returns defaultErrorMap when no options are passed', () => {
-    const errorMap = createErrorMap();
-    expect(errorMap).toBe(defaultErrorMap);
-  });
-
-  test('returns defaultErrorMap when options partially match the default options', () => {
-    const errorMap = createErrorMap({
-      includePath: defaultErrorMapOptions.includePath,
-      issuesInTitleCase: defaultErrorMapOptions.issuesInTitleCase,
-    });
-    expect(errorMap).toBe(defaultErrorMap);
-  });
-
-  test('returns defaultErrorMap when options fully match the default options', () => {
-    const errorMap = createErrorMap(defaultErrorMapOptions);
-    expect(errorMap).toBe(defaultErrorMap);
-  });
-
-  test('does NOT return defaultErrorMap when options diverge from the default options', () => {
-    const errorMap = createErrorMap({
-      ...defaultErrorMapOptions,
-      includePath: false,
-    });
-    expect(errorMap).not.toBe(defaultErrorMap);
-  });
-});
+import { createErrorMap } from './errorMap.ts';
```

Comment:
dev: put all errorMap tests in one place so that we can refer to them as documentation